playground/datastore/departements-regions-france.csv

```diff
@@ -0,0 +1,102 @@
+code_departement,nom_departement,code_region,nom_region
```

Comment:
This may be typically the data that would be embedded in a geojson file.
maybe we can find a dataset that adds some other scalar info? Such as population? That could make nicer examples I think

Reply:
You're right, but I couldn't find anything on data.gouv.fr (I wanted to make sure w'ere actually allowed to use the data). Anyway, I found a dataset on available COVID-19 vaccine doses. Is it okay with you if I add it as a third dataset and do a double join ? Otherwise I'll merge it with this one

Reply:
No worries, it's just for demo purposes. So if you found better, good! But if it's not practical, you can also discard this comment